src/main/java/com/simpletalk/enterprise/SimpleTalkController.java

```diff
@@ -82,9 +93,9 @@ public List<Post> fetchAllPosts() {
 
     /**
      * Fetch a post with the given id.
-     *
+     * <p>
```

Comment:
this was just a change that came with auto format not necessary and can be deleted

Reply:
p tags should be closed in HTML.